src/components/taskList.tsx

```diff
@@ -23,7 +29,8 @@ const TaskList = () => {
       return task;
     });
 
-    setTasks(newTasks);
+    // setTasks(÷newTasks);
+    setTasks(newTasks.sort((b, a) => Number(a.complete) - Number(b.complete)));
```

Comment:
Should always be (a, b) here. Manipulation occurs in the function.
Ex) `newTasks.sort((a, b) => Number(b.complete) - Number(a.complete))`